packages/types/src/codec/Enum.spec.ts

```diff
@@ -4,6 +4,11 @@
 
 import Enum from './Enum';
 
+enum TestEnum {
```

Comment:
We could actually go full-hog ts specs now. (Previously it was problematic since I fed it garbage data - either way, not needed right now)